.travis.yml

```diff
@@ -23,6 +23,19 @@ cache:
   directories:
   - $HOME/.yarn-cache
 
+before_install:
```

Comment:
Hmm.. If you run into merge conflicts later, you'll probably have to rebase off master.